teamwork/api.py

```diff
@@ -90,3 +92,36 @@ def search_users(request):
     response.status_code = status_code
 
     return response
+
+@require_POST
+@login_required
+def log_time(request, project_name):
+    """Log date and time for current user in the project"""
+
+    project_name = unquote_plus(project_name)
+    project = get_object_or_404(Project,
+        name=project_name,
+        owner=request.user)
+
+    # get input date can convert to yyyy-mm-dd format
+    logged_date = request.POST.get('date')
+    logged_date_cor = datetime.datetime.strptime(logged_date, '%d/%m/%Y').strftime("%Y-%m-%d")

```

Comment:
If you change import line on top to `from datetime import datetime, timedelta` then you can use just `datetime.strptime` without writing `datetime` twice.